dataactvalidator/scripts/loadTas.py

```diff
@@ -159,30 +214,38 @@ def existing_id(row, existing):
         Args:
             row: row to check in
             existing: Dict[account_num, List[TASLookup]]
+        Returns:
+            account number for existing row
     """
     for potential_match in existing[row['account_num']]:
```

Comment:
Dumb question, why is this a `for`? As soon as you return it's over? In fact, do we use this function anywhere at all? Can we just get rid of it?

Reply:
It's used by pandas to figure out which rows are updates vs additions: line 209. It basically goes over the list of account numbers found in the database and sees if the incoming rows's account numbers are in said list of account numbers.